src/hal/src/command/mod.rs

```diff
@@ -480,6 +480,46 @@ pub trait CommandBuffer<B: Backend>: fmt::Debug + Any + Send + Sync {
         stride: u32,
     );
 
+    /// Functions identically to `draw_indirect()`, except the amount of draw
+    /// calls are specified by the u32 in `count_buffer` at `count_buffer_offset`.
+    /// There is a limit of `max_draw_count` invocations.
+    ///
+    /// Each draw command in the buffer is a series of 4 `u32` values specifying,
+    /// in order, the number of vertices to draw, the number of instances to draw,
+    /// the index of the first vertex to draw, and the instance ID of the first
+    /// instance to draw.
+    unsafe fn draw_indirect_count(
+        &mut self,
+        _buffer: &B::Buffer,
+        _offset: buffer::Offset,
+        _count_buffer: &B::Buffer,
+        _count_buffer_offset: buffer::Offset,
+        _max_draw_count: u32,
+        _stride: u32
+    ) {
+        unimplemented!("Backend doesn't support draw_indirect_count");
```

Comment:
when porting this to master, let's not forget to remove this default impl

Reply:
Agreed